pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BacklogQuotaManager.java

```diff
@@ -268,4 +271,29 @@ private void disconnectProducers(PersistentTopic persistentTopic) {
 
         });
     }
+
+    private void advanceSlowestMessageDeduplicationCursor(PersistentTopic persistentTopic,
```

Comment:
It's better to return a boolean to represent if the broker continues to disconnect producers. If the slowest cursor is used by the deduplication, we should skip the producer disconnection.